guides/common/modules/ref_system-requirements.adoc

```diff
@@ -40,6 +40,11 @@ endif::[]
 * Administrative user (root) access
 * Full forward and reverse DNS resolution using a fully-qualified domain name
 
+{Project} only supports `UTF-8` encoding.
+No other encoding method is supported.
```

Comment:
```suggestion
```
I'm not sure this is needed if you explicitely say "only utf8 is supported" above.

Reply:
Addressed :+1: